Gopkg.toml

```diff
@@ -97,3 +97,7 @@
   go-tests = true
   non-go = true
   unused-packages = true
+
+[[constraint]]
```

Comment:
Can u move this section ahead `prune` section?

Reply:
I fixed it. Thanks!